youritservices/mock-server-bundle/1.0/config/packages/mock_server.yaml

```diff
@@ -0,0 +1,6 @@
+framework:
+    http_client:
```

Comment:
This looks a little bit unexpected for me. Maybe the bundle should use the `prepend extension` here to set the `http_client` config: https://symfony.com/doc/current/bundles/prepend_extension.html

Else I'm unaware how this will be merged with project custom `http_client` configuration? /cc @fabpot

Reply:
Sorry, my fault. It cannot work, I will revert this PR.

Reply:
@alexander-schranz @fabpot  I'll have a look into using that instead.  The original reason for adding this config in the first place, is that without it, the http_client wasn't being registered as a service when the bundle is added as a dev only dependency, which is the only environment it's intended to be used (it's after all designed to load API mocks for testing, not used in production).  I had presumed originally that the http_client service would be available purely via default  config / behaviour, as the project I was testing it in didn't have anything specifically configured, but this doesn't appear to be the case.